src/python/bot/tasks/trials.py

```diff
@@ -17,37 +17,38 @@
 
 from base import utils
 from datastore import data_types
-from system import environment
 
 
-def setup_additional_args_for_app():
-  """Select additional args for the specified app at random."""
-  if environment.is_engine_fuzzer_job():
-    # Not applicable to engine fuzzers.
-    return
+class Trials:
 
-  app_name = environment.get_value('APP_NAME')
-  if not app_name:
-    return
+  def __init__(self, environment):
```

Comment:
Remove environment arg, add "from system import environment" in this file.

Reply:
Done. But now I also renamed the environment used in the method below as there it needs to be a separate copied test environment now. Calling it also "environment" reduces readability...